src/components/Capture/capture.js

```diff
@@ -252,12 +165,8 @@ class Capture extends Component {
     this.setError('SERVER_ERROR')
   }
 
-  onGlareWarning = () => {
-    this.setWarning('GLARE_DETECTED')
-  }
-
+  // TODO: check this signature
```

Comment:
The signature of the front end errors. For front end errors we don't have warnings at the moment therefore the error will always be of `type: error`. I think we should keep it consistent and use the same signature.